### ISSUE TITLE
VMSS list needs to follow continuation tokens

### ISSUE THREAD

**Issue by AlanSt** [Azure/azure-xplat-cli#2960]
CLI Version: **0.10.0**
OS Type: Linux (Ubuntu)
Installation via:  npm

Mode: **ARM**

Environment: **AzureCloud**

Description:
`azure vmss list <rg>` command doesn't output all VMSS - needs to follow continuation tokens

`azure vmss list-all` command doesn't output all VMSS - needs to follow continuation tokens

Steps to reproduce:
1) Create >50 VMSS
2) Run `azure vmss list <rg>` or 'azure vmss list-all'

**Comment by huangpf** [comment]
@AlanSt Could you please help share part of the ```-vv``` log that shows the continuation tokens? It takes some time for me to repro 50 VMSS. :-)

**Comment by AlanSt** [comment]
Pfft.  Template + loops.
(Will send you logs.)

**Comment by huangpf** [comment]
@amarzavery 
Please refer to: #2961

**Comment by huangpf** [comment]
Will be fixed in the new sprint with ```0.19.0``` upgrade of the underlying compute package.

```
azure vmss list-all
info:    Executing command vmss list-all
data:    ResourceGroupName        Name            ProvisioningState  Location
data:    -----------------------  --------------  -----------------  ----------
data:    HPF629BVS                v629b0ilj       Failed             westus
data:    HPF629BVS                v629b10il       Failed             westus
data:    HPF629BVS                v629b11il       Failed             westus
data:    HPF629BVS                v629b12il       Failed             westus
data:    HPF629BVS                v629b13il       Failed             westus
data:    HPF629BVS                v629b14il       Failed             westus
data:    HPF629BVS                v629b15il       Failed             westus
data:    HPF629BVS                v629b16il       Failed             westus
data:    HPF629BVS                v629b17il       Failed             westus
data:    HPF629BVS                v629b18il       Failed             westus
data:    HPF629BVS                v629b19il       Failed             westus
data:    HPF629BVS                v629b1ilj       Failed             westus
data:    HPF629BVS                v629b20il       Failed             westus
data:    HPF629BVS                v629b21il       Failed             westus
data:    HPF629BVS                v629b22il       Failed             westus
data:    HPF629BVS                v629b23il       Failed             westus
data:    HPF629BVS                v629b24il       Failed             westus
data:    HPF629BVS                v629b25il       Failed             westus
data:    HPF629BVS                v629b26il       Failed             westus
data:    HPF629BVS                v629b27il       Failed             westus
data:    HPF629BVS                v629b28il       Failed             westus
data:    HPF629BVS                v629b29il       Failed             westus
data:    HPF629BVS                v629b2ilj       Failed             westus
data:    HPF629BVS                v629b30il       Failed             westus
data:    HPF629BVS                v629b31il       Failed             westus
data:    HPF629BVS                v629b32il       Failed             westus
data:    HPF629BVS                v629b33il       Failed             westus
data:    HPF629BVS                v629b34il       Failed             westus
data:    HPF629BVS                v629b35il       Failed             westus
data:    HPF629BVS                v629b36il       Failed             westus
data:    HPF629BVS                v629b37il       Failed             westus
data:    HPF629BVS                v629b38il       Failed             westus
data:    HPF629BVS                v629b39il       Failed             westus
data:    HPF629BVS                v629b3ilj       Failed             westus
data:    HPF629BVS                v629b40il       Failed             westus
data:    HPF629BVS                v629b41il       Failed             westus
data:    HPF629BVS                v629b42il       Failed             westus
data:    HPF629BVS                v629b43il       Failed             westus
data:    HPF629BVS                v629b44il       Failed             westus
data:    HPF629BVS                v629b45il       Failed             westus
data:    HPF629BVS                v629b46il       Failed             westus
data:    HPF629BVS                v629b47il       Failed             westus
data:    HPF629BVS                v629b48il       Failed             westus
data:    HPF629BVS                v629b49il       Failed             westus
data:    HPF629BVS                v629b4ilj       Failed             westus
data:    HPF629BVS                v629b50il       Succeeded          westus
data:    HPF629BVS                v629b5ilj       Failed             westus
data:    HPF629BVS                v629b6ilj       Failed             westus
data:    HPF629BVS                v629b7ilj       Failed             westus
data:    HPF629BVS                v629b8ilj       Failed             westus
data:    XPLATTSTVMSSGCREATE3194  xplattestvmss5  Creating           westeurope
data:    HPF629BVS                v629b9ilj       Failed             westus
```
